COMPATIBILITY.md

```diff
@@ -341,7 +341,7 @@ Do not necessarily need any special integration, but are encouraged to use `Once
 
 ## Custom steps
 
-Plugins can also implement custom Workflow steps with specialized behavior.
+Plugins can also implement custom pipeline steps with specialized behavior.
```

Comment:
This is uppercase (you only can implement a "custom Pipeline step" if "Pipeline" is referred to the plugin itself)